services/identity-service/src/vc/vc.service.ts

```diff
@@ -31,10 +31,14 @@ export default class VcService {
         payload: toSign,
         privateJwk: await this.vault.readPvtKey(signerDID),
       });
+      const didDoc = (JSON.parse(did.didDoc as string) as DIDDocument);
       return {
-        publicKey: (JSON.parse(did.didDoc as string) as DIDDocument)
-          .verificationMethod[0].publicKeyJwk,
-        signed: signedJWS,
+        publicKey: didDoc.verificationMethod[0].publicKeyJwk,
+        type: DidService.getKeySignType(didDoc.verificationMethod[0].publicKeyJwk?.crv).signType,
```

Comment:
What is the value that is being returned here?

Reply:
it's the sign type i.e. `JsonWebSignature2020`, `EcdsaSecp256k1Signature2019` based on the curve